test/integration/puppetlabs/general_puppet/general_puppet_int_test.clj

```diff
@@ -51,7 +51,8 @@
      app {:jruby-puppet
           {:max-active-instances num-jrubies}
           :versioned-code
-          {:code-id-command (script-path "echo")}}
+          {:code-id-command (script-path "echo")
+           :code-content-command (script-path "echo")}}
```

Comment:
I don't know if it's worth doing now (or at all), but there's a lot of duplication in these configs, with stuff like the jrubies which aren't relevant for the test. Maybe just something to think about